cmd/quadlet/main.go

```diff
@@ -210,6 +211,66 @@ func enableServiceFile(outputPath string, service *parser.UnitFile) {
 	}
 }
 
+func isSha256Digest(imageName string) bool {
+	if strings.HasPrefix(imageName, "sha256:") {
+		return true
+	}
+
+	// only fully specified digests are matched
+	if len(imageName) != 64 {
```

Comment:
`len("sha256:64chars") != 64`

Reply:
Ugh, I apologize, I have misread this function — the `sha256` check is a short-circuit `true`, not a short-circuit `false`.

But, given that, it is not checking for _image digests_. `isImageID` would be a more accurate name.